t/is-and-extract-ipv6-gua.pl

```diff
@@ -0,0 +1,472 @@
+    "::123.123.123.123",
+    "::2222:3333:4444:5555:6666:123.123.123.123",
+    "::3333:4444:5555:6666:123.123.123.123",
+    "::4444:5555:6666:123.123.123.123",
+    "::5555:6666:123.123.123.123",
+    "::6666:123.123.123.123",
+    "::ffff:192.168.1.1",
+    # Unique local address (ULA)
+    "FC11:2222:3333:4444:5555:6666:7777:8888",
+    "FC11::6666:7777:8888",
+    "FC11::",
+    "FC11:2222::123.123.123.123",
+    "FD11:2222:3333:4444:5555:6666::8888",
+    "FD11:2222:3333:4444:5555::8888",
+    "FD11:2222::8888",
+    "FD11::3333:4444:5555:6666:123.123.123.123",
+    "FD11::4444:5555:6666:123.123.123.123",
+    "FD11::5555:6666:123.123.123.123",
+    "FD11::6666:123.123.123.123",
+    "FD11::8888",
+    # Link local unicast addresses
+    "FE81:2222:3333:4444:5555::7777:8888",
+    "FE81:2222:3333:4444::7777:8888",
+    "FE81::8888",
+    "FE91:2222:3333::7777:8888",
+    "FEA1:2222::7777:8888",
+    "FEB1::7777:8888",
+    # Multicast addresses
+    "FF11::",
+    "FF11:2222::",
+    "FF11:2222:3333::",
+    "FF11:2222:3333:4444::",
+    "FF11:2222:3333:4444:5555::",
+    "FF11:2222:3333:4444:5555:6666::",
+    "FF11:2222:3333:4444:5555:6666:7777::",
+    "FF11:2222:3333:4444:5555:6666:7777:8888",
+);
+
+my @valid_ipv6_global = (
+    "2001::abcd:efAB:CDEF",  # case sensitivity
+    "2001:09:0a:0b:0c:0d:0e:0f",  # leading zeros
+    # Miscelaneous valid GUAs
+    "1111:2222:3333:4444:5555:6666:7777:8888",
+    "1111::3333:4444:5555:6666:7777:8888",
+    "1111::4444:5555:6666:7777:8888",
+    "1111::5555:6666:7777:8888",
+    "1111::6666:7777:8888",
+    "2000::",
+    "2001:DB8:4341:0781::0001",
+    "2001:DB8:4341:0781::1",
+    "2001:DB8:4341:0781::100",
+    "2001:DB8:4341:0781:1111::",
+    "2001:DB8:4341:0781:1111:2222:3333:4444",
+    "2001:DB8:4341:0781::4444",
+    "2fff:ffff:ffff:ffff:ffff:ffff:ffff:ffff",
+    "3000::",
+    "3001:DB8:4341:0781::0001",
+    "3001:DB8:4341:0781::1",
+    "3001:DB8:4341:0781::100",
+    "3001:DB8:4341:0781:1111::",
+    "3001:DB8:4341:0781:1111:2222:3333:4444",
+    "3001:DB8:4341:0781::4444",
+    "3fff:ffff:ffff:ffff:ffff:ffff:ffff:ffff",
+    # IPv4-mapped IPv6 addresses
+    "2111::123.123.123.123",
+    "2111:2222::123.123.123.123",
+    "2111:2222:3333::123.123.123.123",
+    "2111:2222:3333:4444::123.123.123.123",
+    "2111:2222:3333:4444:5555::123.123.123.123",
+    "2111:2222:3333:4444:5555:6666:123.123.123.123",
+    "2111:2222:3333:4444::6666:123.123.123.123",
+    "2111:2222:3333::5555:6666:123.123.123.123",
+    "2111:2222:3333::6666:123.123.123.123",
+    "2111:2222::4444:5555:6666:123.123.123.123",
+    "2111:2222::5555:6666:123.123.123.123",
+    "2111:2222::6666:123.123.123.123",
+    "2111::3333:4444:5555:6666:123.123.123.123",
+    "2111::4444:5555:6666:123.123.123.123",
+    "2111::5555:6666:123.123.123.123",
+    "2111::6666:123.123.123.123",
+    "64:ff9b::123.123.123.123",
+    "64:ff9b:1::123.123.123.123",
+);
+
+my @invalid_ipv6 = (
```

Comment:
Delete this. Validity is already tested in `t/is-and-extract-ipv6.pl`.

Reply:
That tests a different regex.  We need to make sure that the updated regex for global addresses doesn't break anything